raco/backends/sql/catalog.py

```diff
@@ -107,10 +107,12 @@ def _convert_zeroary_expr(self, cols, expr, input_scheme):
 
     def _convert_unary_expr(self, cols, expr, input_scheme):
         input = self._convert_expr(cols, expr.input, input_scheme)
-        if isinstance(expr, expression.MAX):
-            return func.max(input)
-        if isinstance(expr, expression.MIN):
-            return func.min(input)
+
+        fname = expr.__class__.__name__.lower()
```

Comment:
We could also just assume that all the ANSI SQL99 aggregate functions are supported by all SQL backends and translate directly to those without worrying about dialects for now: http://users.atw.hu/sqlnut/sqlnut2-chp-4-sect-2.html. Postgres seems to have pretty good SQL99 conformance (http://www.postgresql.org/docs/9.4/static/features-sql-standard.html), so this may be good enough for now.